docs/manual/forms/examples/camunda7.rst

```diff
@@ -41,7 +41,7 @@ Voeg deze regels toe:
 
 .. image:: _assets/decision_table.png
 
-Rol dan de beslissingsdefinitie uit op Camunda.
+Rol dan de beslisdefinitie uit op Camunda.
```

Comment:
For some reason I couldn't find how to do this in the Camunda docs, took me some time to figure out that you can do this from the Camunda modeler by clicking the rocket icon. Should we maybe mention that here?

Reply:
added link to modeler + pointer to the deploy icon